tests/Microsoft.Identity.Test.Unit.net45/PublicApiTests/PublicClientApplicationTests.cs

```diff
@@ -959,6 +959,131 @@ public void B2CAcquireTokenWithB2CLoginAuthorityTest()
             }
         }
 
+        /// <summary>
+        /// Cache state:
+        ///
+        /// 2 users have acquired tokens
+        /// 1 of them is a guest in another tenant => 1 request for each tenant
+        ///
+        /// There are 3 access tokens, 3 ATs, 3 Accounts but only 2 RT
```

Comment:
>3 access tokens, [](start = 22, length = 16)

Why are there 3 ATs if only 2 accounts? (the test doesn't check the number of ATs). And why only 2 RTs?

Reply:
Great question! There are 3 ATs and 3 accounts. RTs are not scoped by realm (tenantID), so there is a single RT for user who requested ATs from different tenants.